test/parallel/test-https-agent.js

```diff
@@ -36,8 +37,8 @@ server.listen(0, function() {
           rejectUnauthorized: false
         }, function(res) {
           res.resume();
-          console.log(res.statusCode);
-          if (++responses == N * M) server.close();
+          assert.strictEqual(res.statusCode, 200);
+          if (++responses === N * M) server.close();
         }).on('error', function(e) {
```

Comment:
I would change this to `on('error', common.fail)` or `on('error', function(e) { throw e; });` not sure what is the preferred way, looks like both are used in other tests.